dataworks-aws-kickstart-adg.tf

```
@@ -0,0 +1,81 @@
+locals {
+  dataworks_aws_kickstart_adg_pipeline_name = "dataworks_aws_kickstart_adg"
+}
+
+resource "github_repository" "dataworks_aws_kickstart_adg" {
+  name             = "dataworks_aws_kickstart_adg"
```

```suggestion
  name             = "dataworks-aws-kickstart-adg"
```
we use underscores in Terraform resource names, but hyphens in repo names

---

Okay let me make the changes